users/jarred/jarred.h

```diff
@@ -19,6 +19,10 @@
 
 #include "quantum.h"
 
+enum userspace_custom_keycodes {
+  VRSN = SAFE_RANGE               // Prints QMK Firmware and board info
+};
+
 // Use 7 wide characters for keymaps
```

Comment:
Missed this before.

Lines 26-28 shouldn't be needed.

But everything else to me looks good.